Document configuration options for java agent enduser attribute collection (#3982)
open-telemetry/opentelemetry.io: content/en/docs/languages/java/automatic/configuration.md

> **Note**: The property/environment variable names listed in the table are
> still experimental, and thus are subject to change.

### Capturing Enduser Attributes

You can configure the agent to capture
[general identity attributes](/docs/specs/semconv/general/attributes/#general-identity-attributes)
(`enduser.id`, `enduser.role`, `enduser.scope`) from instrumentation libraries
like
[JavaEE/JakartaEE Servlet](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/servlet)
and
[Spring Security](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring/spring-security-config-6.0).

This functionality is disabled by default, with the ability to enable for

> individual attributes.

Reviewer: please call out that they are disabled by default because of privacy concerns, and that people should be careful and verify that they are even allowed to turn this on

> **Note**: The property/environment variable names listed in the table are
> still experimental, and thus are subject to change.

### Capturing Enduser Attributes

You can configure the agent to capture
[general identity attributes](/docs/specs/semconv/general/attributes/#general-identity-attributes)
(`enduser.id`, `enduser.role`, `enduser.scope`) from instrumentation libraries
like
[JavaEE/JakartaEE Servlet](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/servlet)
and
[Spring Security](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring/spring-security-config-6.0).

> > **Note**: Given the sensitive nature of the data involved, this feature is
> > turned off by default while allowing selective activation for particular
> > attributes. Users are urged to thoroughly evaluate each attribute's privacy
> > implications before enabling the collection of the data.

Reviewer 2: ```suggestion
> **Note**: Given the sensitive nature of the data involved, this feature is
> turned off by default while allowing selective activation for particular
> attributes. You must evaluate carefully each attribute's privacy
> implications before enabling the collection of the data.
```

Reviewer 2: Always better to talk directly to the perpetrator. :)
